tests/ts_B_random_query_pass_eqc.erl

```diff
@@ -0,0 +1,150 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+%% @doc A module to test riak_ts basic create bucket/put/select cycle.
+
+-module(ts_B_random_query_pass_eqc).
+-compile(export_all).
+
+%%-ifdef(EQC).
+-include_lib("eqc/include/eqc.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-define(NUMBEROFNODES, 3).
+
+confirm() ->
+    [Node | _] = build_cluster(?NUMBEROFNODES),
+    ?assert(eqc:quickcheck(eqc:numtests(10000, ?MODULE:prop_ts(Node)))),
```

Comment:
Are 10,000 tests really necessary? 100 seems to be quite sufficient.  This current test failed because of `test_timed_out` because riak_test kills tests after 2.5 hours unless you tweak the configuration.